### DISCUSSION TITLE
Where can I customize settings for linkwarden?

### DISCUSSION THREAD

**Discussion by hubsi-k** [tteck/Proxmox#2171] [Q&A]
First of all, thanks for the Linkwarden LXC - works well. But where can I customize settings for linkwarden? In particular, I would like to restrict public registration.

Thanks

**Comment by tteck** [comment]
```
nano /opt/linkwarden/.env
```

**Reply by FalenDemon** [reply]
This should be mentioned on LXC page here: https://tteck.github.io/Proxmox/#linkwarden-lxc
Anyway thanks for your time!
